react-for-atom.js.flow

```diff
@@ -0,0 +1,43 @@
+'use strict';
```

Comment:
This is a large file to have to maintain separately and correctly keep in sync. Is this really the best way?